lib/signed-xml.js

```diff
@@ -551,8 +553,8 @@ SignedXml.prototype.loadSignature = function(signatureNode) {
     this.loadReference(references[i])
   }
 
-  this.signatureValue =
-    utils.findFirst(signatureNode, ".//*[local-name(.)='SignatureValue']/text()").data.replace(/\r?\n/g, '')
+  this.signatureValue = Buffer.from(
```

Comment:
nitpick: `signatureValue` field is still initialized to empty string; can we change since at first glance this appears to be a string.

Reply:
Updated.